src/posts/introducing-xradar/index.md

```diff
@@ -0,0 +1,192 @@
+---
+title: 'Weather Radar Data in Xarray'
+date: '2023-05-16'
```

Comment:
@mgrover1, when you get a moment can you add a `summary` entry (one sentence) here?

Reply:
Done!

Reply:
perfect!

Reply:
Thanks @andersy005 also for the RawHtml update.🙏